docs/ruleset_functions/is_heating_type_fluid_loop(hvac_b.id).md

```diff
@@ -0,0 +1,23 @@
+# is_heating_type_fluid_loop 
+
+**Description:** Returns TRUE if the HVAC system heating system heating type is fluid loop. Returns FALSE if the HVAC system heating system has anything other than fluid loop or if it has more than 1 heating system.   
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the hvac system is modeled with heating type fluid loop.   
+- **hvac_b.id**: The id of the hvac system to evaluate.  
+
+**Returns:**  
+- **is_heating_type_fluid_loop**: Returns TRUE if the HVAC system heating system has fluid loop as the heating type AND only one heating system is associated with the HVAC system. Returns FALSE if the HVAC system has a heating system type other than fluid loop or if it has more than one heating system.   
+ 
+**Function Call:** None  
+
+## Logic:   
+- Check that there is only one heating system associated with the HVAC system: `if Len(hvac_b.heating_system) != 1: is_heating_type_fluid_loop = FALSE`  
```

Comment:
Recommend making it a separate sub-function and remove it from here.

Reply:
Discussed adding differentiation for boiler between hot water and steam with group and decided it was not impactful enough to change.